packages/synthetic-chain/src/lib/vat-status.js

```diff
@@ -49,6 +47,10 @@ const makeSwingstore = db => {
       options: () => kvGetJSON(`${vatID}.options`),
       currentSpan: () =>
         sql.get`select * from transcriptSpans where isCurrent = 1 and vatID = ${vatID}`,
+      terminated: () => {
+        const terminatedIDs = kvGetJSON('vat.terminated');
```

Comment:
The `vat.terminated` key is not always present. When it is not, this throws.

Reply:
What's the repro case?  Is it happening in CI?

I'll file an issue and fix the problem, but I'll get to it faster in some circumstances.

Reply:
https://github.com/Agoric/agoric-sdk/pull/11060#discussion_r1974849553

Reply:
The CI tests in #11060 are all green.

Reply:
They are now, but they were not as of the comment I cited.

Is it not clear that the `vat.terminated` key is not always present?
And that when it is not, this throws?

Reply:
> Is it not clear that the vat.terminated key is not always present?
> And that when it is not, this throws?

It is clear. I was looking for a reproduction case so I could test any fix I come up with. The problem arose because I made a change to the library code, and didn't have a test case, so I was editing the code blindly. I didn't want to do that again.

> They are now, but they were not as of the comment I cited.

I'll see if I can dredge up the historical failure.